lua/lspconfig/util.lua

```diff
@@ -393,4 +383,9 @@ function M.find_package_json_ancestor(startpath)
   return vim.fs.dirname(vim.fs.find('package.json', { path = startpath, upward = true })[1])
 end
 
+--- @deprecated use `vim.fs.find('.git', { path = startpath, upward = true })[1]` instead
+function M.find_git_ancestor(startpath)
+  return vim.fs.find('.git', { path = startpath, upward = true })[1]
```

Comment:
use `vim.fs.dirname` to get parent dir of result  `{ "/Users/mw/workspace/neovim/.git" }`

Reply:
https://github.com/neovim/nvim-lspconfig/pull/3505

Reply:
Considering this is a very common situation, maybe it is better to add parent = true to the filter table

Reply:
You mean to vim.fs.find? But we already have vim.fs.root that does this, I don't think we need to add anything to vim.fs.find.